### DISCUSSION TITLE
Implement miss indicator for spectator

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22836] [Ideas]
The original inspiration for the idea came from the tournament client, where a player's nickname turns red when they miss a note.

I'd like to propose implementing a miss indicator for the spectator mode in multiplayer. The indicator could be displayed on the entire screen (similar to the game [Doom](https://youtu.be/Q4GiCg_m8wA?t=2184)) or by the sides of the gameplay screen (like [in this video](https://youtu.be/SlY6qFmRAPs)) . Alternatively, any other convenient and visually appealing method could be used.
Actually, this could be implemented for tournament client as well to increase visibility of a miss and "dramatic" feelings when you watch a game.

![miss](https://user-images.githubusercontent.com/86934170/224485265-f7b3fb68-daf6-41d0-a0d9-af75d8512cdf.png)

**Comment by Walavouchey** [comment]
i don't see why this should be toggleable, when it could instead just always be a thing when multiple players are visible on the screen

**Reply by Loreos7** [reply]
I guess some of the players may not like this special effect and would ask to have an option to disable it.

**Comment by Axilotl17** [comment]
it should probably be only when combo is broken at a high combo: like how the miss sound doesnt play if you miss many notes in a short span, it only plays when you miss on high-ish combo

**Comment by peppy** [comment]
I don't think this necessarily even needs a toggle. Seems like a cool effect to always have on spectator views to me.
